Detectors/ITSMFT/ITS/macros/test/run_digi.C

```diff
@@ -44,6 +44,7 @@ void run_digi(Int_t nEvents = 10, TString mcEngine = "TGeant3"){
         TGeoManager::Import("geofile_full.root");
 
         // Setup digitizer
+        // Call AliceO2::ITS::DigitizerTask(kTRUE) to activate the ALPIDE simulation
```

Comment:
Probably not so important here in the test macro, but this doesn't need to be a pointer.